### PR TITLE
Fix duplicate imports

### PR DESCRIPTION
**Pull request by renanmav**
Related with #158

### CODE REVIEW

File: src/TypeScriptGenerator.ts [hunk 1]
```diff
@@ -48,11 +48,13 @@ const MODULE_IMPORT_FIELD = "MODULE_IMPORT_FIELD";
 const DIRECTIVE_NAME = "raw_response_type";
 
 export const generate: TypeGenerator["generate"] = (schema, node, options) => {
-  const ast: ts.Statement[] = IRVisitor.visit(
+  let ast: ts.Statement[] = IRVisitor.visit(
```
Comment (sibelius) [inline src/TypeScriptGenerator.ts]: can we avoid `let`?

Comment (renanmav) [reply src/TypeScriptGenerator.ts]: Why?

Comment (maraisr) [reply src/TypeScriptGenerator.ts]: Always eer on `const`. `let` is bug is intent-shift-prone.

**Comment by renanmav** [comment]
For some reason that I don't know yet, `ReaderFragment` imports are not aggregated.

I'll merge this PR for now, investigate then open a new one.

**Comment by alloy** [comment]
:rocket: PR was released in v10.1.3 :rocket:

**Pull request merged**